.changeset/odd-insects-cough.md

```diff
@@ -0,0 +1,5 @@
+---
+'@emotion/styled': minor
+---
+
+fix for typescript error importing @emotion/styled/base when using isolatedModules
```

Comment:
```suggestion
Fix for TypeScript error when importing `@emotion/styled/base` in combination with `isolatedModules` option.
```

Reply:
I've wanted to commit this suggestion through the GitHub interface - but it seems like I don't have sufficient permissions to push to your fork.

You can check how to allow this for the future PRs here - https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork . At the moment just commit this suggestion with a click on a button.

Reply:
done, thanks